docs/commands/csv2po.rst

```diff
@@ -88,34 +92,45 @@ The resultant CSV file has the following layout
 |  C     | Target Language | The msgstr or target language               |
 +--------+-----------------+---------------------------------------------+
 
+
 .. _csv2po#examples:
 
 Examples
 ========
 
-These examples demonstrate the use of csv2po::
+These examples demonstrate the use of csv2po:
+
+.. code-block:: console
```

Comment:
Can we rather use ``.. hightlight:: console`` to set the highlighting for the file.  Adding all these codeblocks actually makes it harder to read the raw text.